### ISSUE TITLE
Adding additional input shows wrong "... event type updated successfully" message

### ISSUE THREAD

**Issue by CarinaWolli** [calcom/cal.com#5245]
### Issue Summary

When adding an additional input field to an event type it shows a toast "... event type updated successfully" when clicking save even though it is not updated. You still need to click save on the top right. 

See here: https://www.loom.com/share/89c5abd6502c4e1098fd9f500c4624da

The pop-up also needs some updates in design. 

Figma Design: 
![Screenshot 2022-10-27 at 13 09 18](https://user-images.githubusercontent.com/30310907/198269060-7b259269-6e13-4868-9a12-6fdfa6e6d7ea.png)

How it currently looks: 
<img width="580" alt="Screenshot 2022-10-27 at 13 11 23" src="https://user-images.githubusercontent.com/30310907/198269418-0e284326-74d7-4ff7-9bc6-c866c9a7b92f.png">

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli Can you give me more details on how you want the design to look at the end of the toast?

**Comment by MuriloAmarals** [comment]
One more thing, what do you want this button to save the additional input to do? Because I thought it makes sense for him to only save momentarily, since the button to save changes to the event type as a whole has not yet been pressed.

**Comment by MuriloAmarals** [comment]
If you think it's a good idea to solve the issue, it is to change the writing in the toast after saving the additional input to "Additional input has been changed. To save the changes click on Save".
@CarinaWolli

**Comment by CarinaWolli** [comment]
@MuriloAmarals I think once we change it to the new design and have a 'create' button instead of 'save' it should be clear that it still needs to be saved (like for all other changes). And we just don't show any toast at all when creating the additional input

**Comment by MuriloAmarals** [comment]
> @MuriloAmarals I think once we change it to the new design and have a 'create' button instead of 'save' it should be clear that it still needs to be saved (like for all other changes). And we just don't show any toast at all when creating the additional input

Ok @CarinaWolli I liked it, I'm going to put my hands on it!!

**Comment by MuriloAmarals** [comment]
![image](https://user-images.githubusercontent.com/87545137/198289434-9480c1f0-a0e3-42e0-9671-b70c4d662151.png)

@CarinaWolli look ate this, I think the checkbox is very user-friendly and accurate than a Dropdown. Don't you agree?

**Comment by MuriloAmarals** [comment]
> ![image](https://user-images.githubusercontent.com/87545137/198289434-9480c1f0-a0e3-42e0-9671-b70c4d662151.png)
> 
> @CarinaWolli look at this, I think the checkbox is very user-friendly and accurate than a Dropdown. Don't you agree?

A dropdown for just a yes or no option doesn't make much sense.

**Comment by CarinaWolli** [comment]
@MuriloAmarals Yes, we can leave the checkbox 👍 Just make sure to have the margins right, otherwise it looks good :)

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli Look what I've got
![image](https://user-images.githubusercontent.com/87545137/198343525-53ba46ee-0a66-4c30-bd63-a3d3ceb60186.png)

**Comment by MuriloAmarals** [comment]
But I am not able to remove Toast, I think it is directly linked to the main button of "Save". If I remove the popup button, it will remove everything apparently, but I'll try something else.

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli Friday I tried to solve the toast problem when saving changes to Additional Input, but no success. I have doubts if it is possible. Do you have any suggestions or can I send the PR just with the new design that is already ready (photo I sent above)?

**Comment by MuriloAmarals** [comment]
@CarinaWolli What do you think about changing the name of the issue to just the css of the Additional Input and then, if possible, open another issue for the toast?

**Comment by CarinaWolli** [comment]
@MuriloAmarals Just mention in the PR description, what exactly your PR fixes. So mentioned this issue and what it fixes of it and what not

**Comment by MuriloAmarals** [comment]
Hey @CarinaWolli I couldn't solve the problem with the toast and another one came up when I went to test the PR. Apparently when I edit an Additional Input it gives an error as in the video below, because to make the design you sent it was necessary to remove the placeholder input. I'll send a print of how it turned out with the placeholder input, what do you think?

https://www.loom.com/share/80f7270ff92c45319a568fd048e790ae

**Comment by MuriloAmarals** [comment]
The design was like this, if you wish I can try to remove the existing placeholder input.
![image](https://user-images.githubusercontent.com/87545137/199626286-f51eeb58-5433-4bf3-806c-9d0c001661bf.png)

**Comment by CarinaWolli** [comment]
@MuriloAmarals I think we can leave the placeholder input like it currently is

**Comment by MuriloAmarals** [comment]
@CarinaWolli Ok!

**Comment by PeerRich** [comment]
this will hopefully be fixed by @hariombalhara with the new form builder
